api/types.proto

```diff
@@ -277,6 +277,41 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum OnFailure {
+		PAUSE = 0;
```

Comment:
Also, it is possible to resume the update with `docker service update servicename` (because the control API endpoint `UpdateService` clears the update status).